.circleci/config.yml

```diff
@@ -1,15 +1,19 @@
+---
 version: 2
 jobs:
   build:
+    working_directory: ~/useradm
     docker:
-      - image: python:3.5
-    working_directory: /home/ubuntu/useradm
+      - image: python:3-alpine
+        environment:
+          ENV: CI
+      - image: dinkel/openldap
```

Comment:
Any benefits to this image over others, e.g. https://github.com/osixia/docker-openldap